package.json

```diff
@@ -43,7 +43,7 @@
 		"db:dev": "run-s db:create:dev db:migrate:dev",
 		"db:test": "run-s db:create:test db:migrate:test",
 		"prepare": "husky install",
-		"precommit": "run-p lint build",
+		"precommit": "run-p lint",
```

Comment:
it is too slow, let's remove it.
ps: app should not rely on this to ensure code in main branch is compilable anway, so it should be safe to remove it.
ps2: the github onpush.yml will do a build as well.